content/en/docs/tasks/administer-cluster/change-default-storage-class.md

```diff
@@ -37,7 +37,9 @@ for details about addon manager and how to disable individual addons.
 
 ## Changing the default StorageClass
 
-1. List the StorageClasses in your cluster: `kubectl get storageclass`
+1. List the StorageClasses in your cluster: 
+
+       kubectl get storageclass
```

Comment:
According to the preview output, you still need to add three backticks for code format. Also, the numbered list is broken, please consider fix it together.